optimism/devnet.go

```diff
@@ -430,10 +430,8 @@ func (d *Devnet) InitChain(maxSeqDrift uint64, seqWindowSize uint64, chanTimeout
 		L2GenesisBlockParentHash:    common.Hash{},
 		L2GenesisBlockBaseFeePerGas: uint642big(1000_000_000),
 
-		OptimismBaseFeeRecipient:    common.Address{0: 0x42, 19: 0xf1}, // tbd
-		OptimismL1FeeRecipient:      d.Addresses.Batcher,
-		L2CrossDomainMessengerOwner: common.Address{0: 0x42, 19: 0xf2}, // tbd
-		GasPriceOracleOwner:         common.Address{0: 0x42, 19: 0xf3}, // tbd
+		L2CrossDomainMessengerOwner: common.Address{0: 0x42, 19: 0xf2},           // tbd
+		L1ERC721BridgeProxy:         common.Address{0: 0x42, 18: 0xa5, 19: 0x7b}, // tbd
```

Comment:
I think this is set inside of `InitDeveloperDeployedAddresses` on the config, there should be a devaddr for it

Reply:
Thanks! Was left over from before the config was filled via that function.